src/index.css

```
@@ -299,10 +305,12 @@
       /*background-color: var(--background-color-alpha);*/
       box-shadow: 0 0 0 0 var(--background-color-alpha);
     }
+
     50% {
       /*background-color: var(--background-color-alpha);*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)